web/src/containers/Notification.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Snackbar from '@material-ui/core/Snackbar';
+
+import { processQueue, closeNotification } from '../actions/notification';
+
+class Notification extends Component {
+  handleClose = (_, reason) => {
+    if (reason === 'clickaway') return;
+    this.props.closeNotification();
+  };
+
+  handleExited = () => {
+    this.props.processQueue();
```

Comment:
And then pop the queue only after the previous notification has transitioned away.  Slick!